Source/Hammer.spoon/init.lua

```diff
@@ -0,0 +1,95 @@
+--- === Hammer ===
```

Comment:
While I like the idea of `Hammer.spoon` I think this is too generic a name :)

IMO this should probably be slimmed down to just being a config auto-reloader, and called `ConfigAutoReload.spoon` or similar.